### ISSUE TITLE
Identify and compile and PM roster

### ISSUE THREAD

**Issue by heyitsalexander** [hackforla/VRMS#1274]
### Overview
We need to be able to elevate PMs access levels to that of project-level admins.  That way, we PMs can add/edit meeting times of their individual projects without having to put in a request to the VRMS team. This will allow PMs to start utilizing and incorporating VRMS into their workflow and the VRMS team will be able to obtain valuable feedback from a user's perspective.

### Action Items
1. Bonnie will make a list of active projects and share it with google Drive
2. Bonnie will permission active PMs for project level admins.
3. VRMS will engage the ones where the PMs status is unknow and update the spreadsheet. 
4. Reconvene Feb 6 for a lunch 'n learn to show PMs to teach how to change meeting time.

### Resources/Instructions
- [VRMS: Active projects as of 2023-01 spreadsheet](https://docs.google.com/spreadsheets/d/1ZSyCkO3XgD19EzmpLPkmz196vpwNnjjNMZZqXUV8Wqw/)

**Comment by heyitsalexander** [comment]
@jbubar @bonniewolfe I just tried testing if the meeting time data on VRMS is being accurately reflected on the website. I added a new "testing" meeting time on VRMS last night and it's still not showing on the website as of this morning.  Screenshots attached. Let me know what you think.



<img width="899" alt="Screen Shot 2023-01-24 at 9 05 33 AM" src="https://user-images.githubusercontent.com/39230947/214362540-c8fd5aa2-7a06-434d-9060-3abb2bc6d1ec.png">
<img width="513" alt="Screen Shot 2023-01-24 at 9 05 28 AM" src="https://user-images.githubusercontent.com/39230947/214362559-0588a7c1-4177-454c-a6b3-df3503099a1c.png">

**Comment by ExperimentsInHonesty** [comment]
- I created the [Active projects as of 2023-01 spreadsheet](https://docs.google.com/spreadsheets/d/1ZSyCkO3XgD19EzmpLPkmz196vpwNnjjNMZZqXUV8Wqw)
- added all the PMs that I knew about to VRMS
- Indicated the project's where I can't do that yet in red
- and Messaged the three PMs I can't find in VRMS at all
- Bonnie [2023-01-29 19:43 AM PST](https://hackforla.slack.com/archives/C010LNXH2JY/p1675014196907179)
- [@Bishr Aboobaker](https://hackforla.slack.com/team/U03VDRGD34P) [@Christian Butcher](https://hackforla.slack.com/team/U04GYRTTBHD) [@Ebi](https://hackforla.slack.com/team/U031EU2E6LB) you seem to be missing from VRMS.  I am trying to add permissions for you to edit project meeting data.  Please create a profile in VRMS by visiting this link https://vrms.io/newProfile and then let us know by posting a comment on this issue https://github.com/hackforla/VRMS/issues/1274

**Comment by ExperimentsInHonesty** [comment]
@heyitsalexander I will check in with the dev lead today, I don't see the changes on the website still.

- I opened a new ticket on the github.com/hackforla/website repo about this issue https://github.com/hackforla/website/issues/3890
  - FYI, the api from VRMS is working fine, there is just a glitch on hackforla.org website.  The data is showing up correctly on the https://www.hackforla.org/events/ page.

**Comment by cbutcher318** [comment]
@bonniewolfe I went to the link posted on Slack and created a profile. Let me know if you need anything else.

**Comment by bishrfaisal** [comment]
@bonniewolfe I went to the link and created a VRMS profile. Thank You.

**Comment by Biuwa** [comment]
@ExperimentsInHonesty I created a profile. Thank you for bringing it to my notice.

**Comment by edwardsarah** [comment]
https://github.com/hackforla/website/issues/3925 Submitted this issue to add me as a current PM to the Guides Team project profile. Linking here as Bonnie asked me to on slack.

**Comment by ExperimentsInHonesty** [comment]
Please provide update
- Progress: I have processed all the changes made above and below (see comments with thumbs up)
  - See the [spreadsheet](https://docs.google.com/spreadsheets/d/1ZSyCkO3XgD19EzmpLPkmz196vpwNnjjNMZZqXUV8Wqw/edit#gid=0) for details of outstanding items
- Blockers: Outstanding items
  - VRMS problem: Food Oasis does not show up in the list of projects, but when I try to add it, it says the name is already taken.  So I can't add the project to any of these users
  - People need sorting out 
     - Project: VRMS 
        - Julia has potentially two names in the VRMS.  
        - Amanda does not have an account in VRMS
     - Project: UI/UX waiting on the community of practice leads to get back to me
     - Project: Expunge Assist waiting on the community of practice leads to get back to me 
- Availability: 30 min (ill check in next Monday 2023-02-06)
- ETA: Depends on when people get back to me, but I estimate that it will be resolved by end of meeting on Monday 02-06.

Note:
Once Bonnie's updating of Project PMs is done, then the fix has to happen on the website so that the data is showing up on the website: https://github.com/hackforla/website/issues/3890

**Comment by randelbrot** [comment]
@ExperimentsInHonesty I submitted website [#3929](https://github.com/hackforla/website/issues/3929) and VRMS #1284 for the Home Unite Us project page updates, per your request

**Comment by Aparna1Gopal** [comment]
@ExperimentsInHonesty As requested on Slack:
The UI/UX CoP Co-Leads list is up to date on the website. The issue was tracked [here](https://github.com/hackforla/website/issues/3691).

**Comment by ExperimentsInHonesty** [comment]
2023-02-04 8:45am pst: I did another round of checking, the below is the update, and previous updates are hidden
- Progress: I have processed all the changes made above and below (see comments with thumbs up)
  - See the [spreadsheet](https://docs.google.com/spreadsheets/d/1ZSyCkO3XgD19EzmpLPkmz196vpwNnjjNMZZqXUV8Wqw/edit#gid=0) for details of outstanding items 2023-02-04
- Blockers: Outstanding items
  - VRMS problem: Food Oasis does not show up in the list of projects, but when I try to add it, it says the name is already taken.  So I can't add the project to any of these users
  - People need sorting out 
     - Project: VRMS 
        - Julia has potentially two names in the VRMS.  
        - Amanda does not have an account in VRMS
     - Project: Expunge Assist waiting on Sheron to signup for an account in VRMS
- Availability: 30 min (ill check in next Monday 2023-02-06)
- ETA: Depends on when people get back to me, but I estimate that it will be resolved by end of meeting on Monday 02-06.

Note:
- Once Bonnie's updating of Project PMs is done, then the fix has to happen on the website so that the data is showing up on the website: https://github.com/hackforla/website/issues/3890

**Comment by sheronvirani21** [comment]
@ExperimentsInHonesty I have submitted for a VRMS account as requested

**Comment by ExperimentsInHonesty** [comment]
I have completed the spreadsheet and identified all the pms.

**Comment by Rabia2219** [comment]
@ExperimentsInHonesty submitted the change request: [Update Project Profile: Youth Justice Nav](https://github.com/hackforla/website/issues/3949)
